### PR TITLE
Add option to select specific countries

### PR DESCRIPTION
**Pull request by mumblebaj**
Added an option for user to select a specific list of countries to display instead of the full list of countries

image

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mmm-olympicgames",
-  "version": "1.2.0",
+  "version": "1.2.1",
```
Comment (fewieden) [inline package.json:3]: It should match the version in the `package-lock.json` file

File: node_helper.js [hunk 2]
```diff
@@ -18,14 +18,32 @@ module.exports = NodeHelper.create({
...
+            this.getCountryMedals(this.config);
             setInterval(() => {
-                this.getCountryMedals();
+                this.getCountryMedals(this.config);
             }, this.config.reloadInterval);
         }
     },
 
-    async getCountryMedals() {
+    formatResults: function(payload, countries) {
```
Comment (fewieden) [inline node_helper.js]: You are not formatting the data, I would rename the function name to `filterCountryList`.

Additionally, I think this should be placed in the `MMM-OlympicGames,js` file, which is responsible for displaying the correct data.

File: README.md [hunk 3]
```diff
@@ -49,6 +49,7 @@ To set a global config you have to set the value in your config.js file inside t
 | `title` | `'Olympic Winter Games 2022'` | The title above the medal table. |
 | `reloadInterval` | `1800000` (30 mins) | How often should the data be fetched. |
 | `provider` | `'bloomberg'` | Specify the data source. Possible options: `'bloomberg'` and `'nbc'`. |
+| `countryList` | `[ "NO", "AU", "US", "DE", "FR", "CA" ]` | Specify a list of country codes to display (alpha-2 code). |
```
Comment (fewieden) [inline README.md:52]: The default of this value is undefined. However, I think your example is valuable and should be moved inside the description of the config option.

I also think the `maxRows` option should be ignored if the `countryList` is specified.

**Comment by fewieden** [comment]
@mumblebaj thanks for your contribution, I left some comments

File: README.md (hunk 3 above)
Comment (mumblebaj) [reply README.md:52]: Awesome. Will take a look at these and submit a new PR

File: node_helper.js (hunk 2 above)
Comment (mumblebaj) [reply node_helper.js]: I thought it might be easier to just pass the filtered list to the MMM-OlympicGames.js instead of filtering at run time.

**Pull request closed without merging**